docs/sidebars.ts

```diff
@@ -434,7 +434,7 @@ const sidebars: SidebarsConfig = {
         {
           type: 'link',
           label: 'Python API',
-          href: 'https://mlflow.org/docs/latest/api_reference/python_api/index.html',
+          href: `${process.env.DOCS_HOST}${process.env.DOCS_BASE_URL}api_reference/python_api/index.html`,
```

Comment:
```suggestion
          href: `${process.env.API_DOCS_PREFIX}api_reference/python_api/index.html`,
```

this might be simpler and harder to make a mistake.